CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Added BLAME function.
+- Removed redundant commit information from BLAME results.
```

Comment:
Please, use the format from https://keepachangelog.com/en/1.0.0/ .

Reply:
:+1: